src/BaGet.UI/src/App.tsx

```diff
@@ -61,6 +61,7 @@ class App extends React.Component<RouteComponentProps, IAppState> {
                 className="form-control"
                 autoComplete="off"
                 placeholder="Search packages..."
+                data-testid="input-field"
```

Comment:
What do you think of a name like `search-field` to make its purpose a little clearer?

Reply:
perfect xD